projects/pedal_board/src/brake_data.c

```diff
@@ -0,0 +1,20 @@
+#include "log.h"
+#include "pedal_data_tx.h"
+#include "pedal_events.h"
+#include "soft_timer.h"
+
+// NEED TO CHANGE!!!!!!
+#define UPPER_BRAKE_VALUE 1273
+#define LOWER_BRAKE_VALUE 295
+#define MULTIPLYER 4096
+
+StatusCode get_brake_data(PedalDataStorage *storage, int16_t *position) {
+  status_ok_or_return(ads1015_read_raw(storage->storage, storage->brake_channel, position));
+  int32_t percent = UPPER_BRAKE_VALUE - LOWER_BRAKE_VALUE;
```

Comment:
rename `percent`  to `range`

Reply:
This module is taking another module's `storage` as input, that's weird...  You need to access ads1015's storage since you're using only one instance of `ads_1015`. It would make sense if you had a `PedalData` module and had that module initialize the channels on `ads1015`,  then have that module provide a global storage to the ads1015. Then over here you could get that storage, and call ads1015_read_raw with that storage. This is a bit messy right now.